src/pages/abs/[id]/abstract.tsx

```diff
@@ -110,7 +110,7 @@ const AbstractPage: NextPage<IAbstractPageProps> = (props: IAbstractPageProps) =
                         params={createQuery('orcid', orcid)}
                         aria-label={`author "${author}", search by orKid`}
                       >
-                        <OrcidActiveIcon fontSize={'large'} />
+                        <OrcidActiveIcon fontSize={'large'} mx={1} />
```

Comment:
I haven't tested this, but do these now show up in the tabindex? 
I wasn't able to tab into these icon-links.  We may want to change to `<IconButton>` and restyle.  But if not we'll need to add some extra attributes to make more accessible

Reply:
@thostetler Which browser? I see them as `<a>` tag and in Chrome I was able to tab through them. Firefox on Mac requires turning it on in the settings and Safari also has tabbing through links turned off by default.

Reply:
Nevermind after testing these look good :+1: